Vagrantfile

```diff
@@ -95,6 +95,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       app.vm.synced_folder "src/rf/apps", "/opt/app/apps"
     else
       app.vm.synced_folder "src/rf", "/opt/app/"
+      if File.directory?("#{ENV['HOME']}/.aws")
+        app.vm.synced_folder "#{ENV['HOME']}/.aws", "/home/vagrant/.aws/"
+      else
+        puts "An AWS config profile is needed for the application to function."
+        puts "Run `aws configure` to get started."
```

Comment:
I would probably adjust the `aws` command to `aws configure --profile rf-dev` so that the defaults work out of the box.